core/genesis.go

```diff
@@ -57,6 +57,7 @@ func WriteGenesisBlock(chainDb ethdb.Database, reader io.Reader) (*types.Block,
 			Code    string
 			Storage map[string]string
 			Balance string
+			Nonce   string
```

Comment:
I never answered the question, because I don't know about best practices. Please modify as you see fit :)